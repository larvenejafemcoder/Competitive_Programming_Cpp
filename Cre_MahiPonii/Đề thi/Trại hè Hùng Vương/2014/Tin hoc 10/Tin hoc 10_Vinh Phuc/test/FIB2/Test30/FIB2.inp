84
41 88667594
32 1701827
42 138691061
33 2209701
8 9
36 4164985
1 1
8 25
8 23
27 307757
5 8
9 52
10 36
24 2616
35 1449043
38 49569711
44 849633122
14 346
25 101852
18 34
4 4
45 236850026
3 1
22 16236
35 820068
27 160775
38 8712843
33 388912
10 50
31 846936
11 50
18 3671
24 10227
16 1564
0 1
36 401749
33 1213681
30 448012
28 314110
43 649729060
21 1691
42 275825840
6 9
44 1018588236
3 3
24 48086
34 5748584
3 3
24 20632
36 21684117
36 10274332
20 7040
41 18596060
27 222893
6 1
44 792129903
17 1901
36 14198158
3 3
12 192
16 291
3 2
36 15308574
22 4042
22 10327
28 385104
32 2837612
21 12416
39 69207864
25 83175
4 2
42 175987782
5 8
11 113
19 2221
5 5
19 989
15 86
35 7154321
34 2371328
26 150681
27 146657
32 424888
32 3407498
