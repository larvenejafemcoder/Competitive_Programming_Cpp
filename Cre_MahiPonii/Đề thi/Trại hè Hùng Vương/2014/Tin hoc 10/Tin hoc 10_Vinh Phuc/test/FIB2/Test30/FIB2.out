33868007
650040
52975271
844031
4
1590883
0
10
9
117553
3
20
14
999
553485
18933945
324530975
132
38904
13
2
90468660
0
6202
313238
61411
3328010
148551
19
323501
19
1402
3906
598
1
153455
463585
171125
119979
248174418
646
105356096
4
389066086
1
18367
2195764
1
7881
8282596
3924446
2689
7103063
85138
1
302566700
726
5423214
1
73
111
1
5847355
1544
3945
147097
1083871
4743
26435052
31770
1
67221351
3
43
848
2
378
33
2732708
905767
57555
56018
162293
1301549
